crates/swc_ecma_transforms_base/tests/ts-resolver/deno/lint/705/output.ts

```diff
@@ -1,4 +1,7 @@
 import { Foo__1 } from "./foo.ts";
-function _bar__1(...Foo__2: Foo__2) {
```

Comment:
I think the test suite could use improvement if regressions like this can happen.

Maybe instead of using numbers here the tests should instead allow using unique values (which may be numbers) to link identifiers instead of the context number. So for the test to pass, the identifiers in the output just have to be the same instead of having to have a specific syntax context number. This would also reduce a lot of churn in the test output.

Reply:
I'm not against it, but it sounds like a lot of work